devops/Makefile

```diff
@@ -56,108 +58,60 @@ define DB_SERVICE_NAME
 "patroni-master"
 endef
 
-define FORMSFLOW_FORMS_NAME
-"forms-flow-forms"
-endef
-
-define FORMSFLOW_BPM_NAME
-"forms-flow-bpm"
-endef
-
-define FORMSFLOW_API_NAME
-"forms-flow-api"
-endef
-
-define FORMSFLOW_WEB_NAME
-"forms-flow-web"
+define FORMS_NAME
+"forms"
 endef
 
 define HAMONGO_DB_TEMPLATE_PATH
-"openshift/forms-flow-ai/database/mongo-ha.yml"
+"openshift/database/mongo-ha.yml"
 endef
 
 define HAMONGO_DB_PARAM_PATH
-"openshift/forms-flow-ai/database/mongo-ha-param.yml"
-endef
-
-define SECRETS_PATRONI_DB_TEMPLATE_PATH
-"openshift/forms-flow-ai/secrets/ffa-patroni-secrets.yml"
-endef
-
-define CREATE_FFA_DB_TEMPLATE_PATH
-"openshift/forms-flow-ai/database/create-ffa-db.yml"
-endef
-
-define FFA_BPM_DB_NAME
-"FFA_BPM_DB"
-endef
-
-define FFA_API_DB_NAME
-"FFA_API_DB"
+"openshift/database/mongo-ha-param.yml"
 endef
 
 define SECRETS_TEMPLATE_PATH
-"openshift/forms-flow-ai/secrets/secrets.yml"
+"openshift/secrets.yml"
 endef
 
 define SECRETS_PARAM_PATH
-"openshift/forms-flow-ai/secrets/secrets-param.yml"
-endef
-
-define FORM_AI_BUILD_TEMPLATE_PATH
-"openshift/forms-flow-ai/build.yml"
+"openshift/secrets-param.yml"
 endef
 
-define FORM_AI_FORMS_DEPLOY_TEMPLATE_PATH
-"openshift/forms-flow-ai/forms-deploy.yml"
+define FORMS_BUILD_TEMPLATE_PATH
```

Comment:
All these `FORMS_???` seems to be used only once. All other make commands are using it alongside the command. Can we follow the same here?